app/src/main/java/com/cvdevelopers/baseapplication/template_generated_classes/TemplateFragment.kt

```diff
@@ -0,0 +1,42 @@
+package com.cvdevelopers.baseapplication.template_generated_classes;
+
+import com.cvdevelopers.baseapplication.R
+import com.cvdevelopers.baseapplication.baseclasses.BaseFragment
+import com.cvdevelopers.baseapplication.baseclasses.BaseViewModel
+import com.cvdevelopers.baseapplication.template_generated_classes.model.TemplateDisplayData
+import com.cvdevelopers.baseapplication.utils.ViewSubscriptionManager
+import javax.inject.Inject
+import kotlinx.android.synthetic.main.template_fragment.*
+
+class TemplateFragment : BaseFragment() {
+
+    @Inject
+    lateinit var viewModel: TemplateViewModel
+
+    override fun getResourceLayoutId() = R.layout.template_fragment
+
+    override fun initializeSubscriptions(subscriptionManager: ViewSubscriptionManager) {
+        subscriptionManager.apply {
```

Comment:
why do an apply when it is only doing one method on the subscription manager? It makes is pretty hard to read at first glance.